setup.py

```diff
@@ -54,6 +54,5 @@
         "Framework :: Sphinx",
         "Framework :: Sphinx :: Extension",
     ],
-    namespace_packages=["sphinxcontrib"],
```

Comment:
Is it worth to change the packages value further up to explicit values?

Reply:
Could be but I wanted the change to be minimal, so it has a chance to get merged quickly upstream.